trieste/acquisition/function.py

```diff
@@ -888,9 +889,35 @@ class ExpectedHypervolumeImprovement(SingleModelAcquisitionBuilder):
     follows :cite:`yang2019efficient`
     """
 
+    def __init__(
+        self,
+        ref_point_specification: Sequence[float]
+        | TensorType
+        | Callable[[TensorType], TensorType] = get_reference_point,
+    ):
+        """
+        :param ref_point_specification: this method is used to determine how the reference point is
+            calculated. If a Callable function specified, it is expected to take existing pareto
+            front and return a reference point with shape [D] (D represents number of objectives).
+            If the Pareto front location is known, this arg can be used to specify a fixed
+            reference point in each bo iteration. A dynamic reference point updating strategy is
+            used by default to set a reference point according to the datasets.
+        """
+        if callable(ref_point_specification):
+            self._ref_point_specification = ref_point_specification
```

Comment:
May be worth giving an explicit type here (but ok not to if you don't get any mypy errors after the suggested change below)

```suggestion
            self._ref_point_specification : TensorType | Callable[[TensorType], TensorType] = ref_point_specification
```